src/registrar/models/very_important_person.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from .utility.time_stamped_model import TimeStampedModel
+
+
+class VeryImportantPerson(TimeStampedModel):
+
+    """"""
+
+    email = models.EmailField(
+        null=True,
+        blank=True,
+        help_text="Email",
+        db_index=True,
+    )
+
+    user = models.ForeignKey(
+        "registrar.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="verifiedby_user",
+    )
```

Comment:
(Optional) Maybe this field should be called something like "approving_user" or "authorizing_user"? If not, I think more information should exist about the relationship this object has (like a description on /admin). Its a little ambiguous if you aren't privy to the purpose of this table

Reply:
Requestor

Reply:
Requestor is great